### DISCUSSION TITLE
Feasibility in Random Vibration Analysis

### DISCUSSION THREAD

**Discussion by Krystalfang** [idaholab/moose#22335] [Q&A General] [answered]
Dear experts，
I've been trying to do random vibration analysis with moose recently，specific description is：apply an acceleration power spectral density function to the sample, under given conditions, through the acceleration PSD response curve of the test sample at its typical position.
I would like to ask, can MOOSE fulfill this requirement?
Thanks,
Krystal

**Comment by GiudGiud** [comment]
@bwspenc  @cbolisetti

**Comment by cbolisetti** [comment, marked as answer]
Hi @Krystalfang, currently, we can only perform response history analyses in MOOSE. Unfortunately, we cannot perform frequency-domain simulations (or eigen value analysis), which would be needed for RVA.
